src/commands/openWorkflowStepLogs.ts

```diff
@@ -0,0 +1,22 @@
+import * as vscode from "vscode";
+import {WorkflowStepNode} from "../treeViews/workflows/workflowStepNode";
+
+type WorkflowStepCommandArgs = Pick<WorkflowStepNode, "job" | "step" | "gitHubRepoContext">;
+
+export function registerOpenWorkflowStepLogs(context: vscode.ExtensionContext) {
+  context.subscriptions.push(
+    vscode.commands.registerCommand("github-actions.step.logs", async (args: WorkflowStepCommandArgs) => {
+      const job = args.job.job;
+      let url = job.html_url ?? "";
+      const stepName = args.step.name;
+
+      const index = job.steps && job.steps.findIndex(step => step.name === stepName) + 1;
```

Comment:
We want to `+1` at the end because findIndex will get a 0 index based number.  And the first step will be represented in the URL as 1.

Reply:
You could add a comment about the 1 offset here